homework.md

```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A scripting or programming language that allows you to implement complex features on web pages 
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1.Git is a version control system that lets you manage and keep track of your source code history. 
+2.GitHub is a cloud-based hosting service that lets you manage Git repositories. 
+3.If you have open-source projects that use Git, then GitHub is designed to help you better manage them.
 
```

Comment:
You've got a great list here of the features that Git and Github provide. 👍

Can you describe what problems these features solve? What problem does a cloud-based hosting service solve? What management problem does Git solve?